cypress/support/step-definitions/step-sequence-steps.js

```diff
@@ -0,0 +1,27 @@
+import {
+  stepSequence, stepSequenceElement, stepSequenceItemIndicator, ariaLabel,
+} from '../../locators/step-sequence';
+
+const ARIA_LABEL = 'Step 1 of 5';
+
+Then('orientation is set to {string}', (orientation) => {
+  stepSequence().should('have.attr', 'orientation', orientation);
+  stepSequenceElement().should('have.attr', 'orientation', orientation);
+});
+
+Then('indicator {string} is set', (indicator) => {
+  cy.wait(100); // required because iframe content is changed
```

Comment:
I was wondering, could we figure out something to use another wait, not limited in time -> cy.wait('@getStorybook') ?

Reply:
storybook element exists all time, only value or content of that element is changed, can be add to do that for maintenance task after finish faze1